packages/contracts/test/helpers/test-runner/test-runner.ts

```diff
@@ -72,18 +75,40 @@ export class ExecutionManagerTestRunner {
           codeHash: NON_NULL_BYTES32,
           ethAddress: '$OVM_PROXY_EOA',
         },
+        [predeploys.OVM_SafetyCache]: {
+          codeHash: NON_NULL_BYTES32,
+          ethAddress: '$OVM_SAFETY_CACHE',
+        },
+        $OVM_SAFETY_CHECKER: {
+          codeHash: NON_NULL_BYTES32,
+          ethAddress: '$OVM_SAFETY_CHECKER',
+        },
+        // Address of hardhat/console.sol, comes in handy sometimes
+        ['0x000000000000000000636F6e736F6c652e6c6f67']: {
```

Comment:
fun fact, this is ascii for `console.log`

Reply:
Could be a constant like `HEX_CONSOLE_LOG`